boot/zephyr/Kconfig

```diff
@@ -364,6 +370,50 @@ config BOOT_WAIT_FOR_USB_DFU
 	  for USB DFU to be invoked. Please note DFU always updates the
 	  slot1 image.
 
+config BOOT_USB_DFU_WAIT_DELAY_MS
+	int
+	depends on BOOT_USB_DFU_WAIT
+	default 12000
+	help
+	  The duration to wait for USB DFU to be invoked.
+
+config BOOT_USB_DFU_GPIO
+	bool "Use GPIO to detect whether to trigger DFU mode"
+	select USB
+	select USB_DFU_CLASS
+	select IMG_MANAGER
+	help
+	  If y, MCUboot uses GPIO to detect whether to invoke USB DFU.
+	  Please note DFU always updates the slot1 image.
```

Comment:
I do not think that this comment is needed here.

Reply:
Previously, this blurb was only in the `BOOT_WAIT_FOR_USB_DFU` config option: where should it go after a second way to enable USB DFU is added?

Reply:
I mean the "Please note DFU always updates the slot1 image".
The problem with this comment is that you may change where the image will be loaded without knowing this comment exists, which will make it misleading. For example I am doing something like this here https://github.com/zephyrproject-rtos/zephyr/pull/29974.

Reply:
Yeah, I meant that as well. Do you mean that I should just delete the line here (and in `BOOT_USB_DFU_WAIT`)?

Reply:
Yes, I think so. The option does not affect which Image is selected for upload.